devenv/private-cluster.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
It seems incorrect to test this way here. We should be relying on the exact k8s manifests that rp uses to guarantee it, not creating arbitrary manifests that are meaningless in the context of the real application.

Reply:
that's a good point - I just took this from the existing e2e framework. I imagine it's a copy of what RP uses anyway, but I'll look to see what I can do to synchronize the two.

Reply:
I think the smartest way to go about this will just be to use some sort of embedded file system that we can import from RP, but given how many other changes we're planning for this add-on, I'm gonna regard that as out of scope and a test enhancement that we can add later as we start changing stuff up on the testing side. Just want to have SOME tests at the moment.